python/amici/sbml_import.py

```diff
@@ -869,6 +870,133 @@ def _process_time(self) -> None:
+                        # if the delay is not a fixed time, so is used here
+                        # also to represent unsupported delays.
+                        raise TypeError
+                except TypeError:
+                    raise SBMLException('Events with execution delays are '
+                                        'currently not supported in AMICI.')
+            if not event.getUseValuesFromTriggerTime():
+                raise SBMLException('Events with '
+                                    '"use_values_from_trigger_time == False" '
+                                    'are currently not supported in AMICI.')
+            if event.getPriority() is not None:
+                raise SBMLException(f'Event {event_id} has a priority '
+                                    'specified. This is currently not '
+                                    'supported in AMICI.')
+
+            trigger_sbml = event.getTrigger()
+            if trigger_sbml is None:
+                # TODO raise SBMLException? Or just ignore?
+                continue
+            if trigger_sbml.getMath() is None:
+                # TODO untested, find SBML test suite case
+                logger.warning(f'Event {event_id} trigger has no trigger '
+                               'expression. Ignoring event...')
+                continue
+            if not trigger_sbml.getPersistent():
+                raise SBMLException(
+                    f'Event {event_id} trigger is persistent. '
+                    'Non-persistent / delayed / prioritized events are '
+                    'currently not supported in AMICI.'
+                )
+            # FIXME check what AMICI supports by default
+            # True means event does not executed if triggered at time == 0
+            # False means it does.
+            if not trigger_sbml.getInitialValue():
+                raise SBMLException(
+                    f'Event {event_id} has a trigger that has an initial '
+                    'value of False. This is currently not supported in AMICI.'
+                )
+            if not trigger_sbml.getInitialValue():
+                raise SBMLException(
+                    f'Event {event_id} has a trigger that has an initial '
+                    'value of False. This is currently not supported in AMICI.'
+                )
+            trigger_sym = self._sympy_from_sbml_math(trigger_sbml)
+            trigger = _parse_event_trigger(trigger_sym)
+
+            # FIXME?: Currently, all event assignment targets must exist in
+            # self.symbols[SymbolId.SPECIES]
+            state_vector = list(self.symbols[SymbolId.SPECIES].keys())
+
+            bolus = [sp.Symbol('0') for _ in state_vector]
+
+            event_assignments = event.getListOfEventAssignments()
+            if not len(event_assignments):
+                # ignore events with no event assignments
+                # TODO raise exception?
```

Comment:
I think continue is fine here.

Reply:
Yes, makes sense to allow void assignments, as we sometimes will just want to get some read out.